guides/admin-guide.rst

```diff
@@ -13,10 +13,10 @@ resource manager on a cluster.
     in this guide may change with regularity.
 
     This document is in DRAFT form and currently applies to flux-core
-    version 0.45.0.
+    version 0.50.0.
```

Comment:
Whoa! Are we really nearing 50?! I hope we can get that bugfix in for zeromq - I would be a very happy dinosaur.

Reply:
I've proposed it and tentatively added it to the release notes for 50.